### PR TITLE
PBI 27 28 29 read edit delete material content

### PR DESCRIPTION
**Pull request by Zsaschz**

### CODE REVIEW

File: src/resources/app/pages/course/CourseMaterial.vue [hunk 1]
```diff
@@ -61,17 +74,18 @@ export default {
...
       try {
         let response = await getCourseMaterialById(this.$props.materialId);
-        if (response.data.topic === null){
-          response.data.topic = {title: ''}
+        if (response.data.topic === null) {
+          response.data.topic = { title: "" };
         }
         this.material = response.data;
       } catch (error) {
+        console.log(error);
```
Comment (IlmaMannix) [inline src/resources/app/pages/course/CourseMaterial.vue]: please remove this

**Review by IlmaMannix — COMMENTED** [review]
please resolve conflict first

**Review by IlmaMannix — APPROVED** [review]
LGTM, thanku thann 👍

**Review by fxdros — APPROVED** [review]
Looks good! Thanks, fella!

**Pull request merged**